src/concrete/ml/pandas/_processing.py

```diff
@@ -189,10 +297,15 @@ def pre_process_dtypes(pandas_dataframe: pandas.DataFrame) -> Tuple[pandas.DataF
                 "supported."
             )
 
+    # TODO: Should all non-integers columns be considered by the schema if not None ? Currently,
```

Comment:
should we raise an error/warning if all non-integer columns from the data-frame were not covered by the given schema ?

Reply:
What would happen if they are missing from the given schema?

Reply:
they are automatically computed

Reply:
I'm just wondering because if could be an easy mistake to forget to put some columns, but no error will be raised